packages/fxa-auth-server/test/remote/password_change_tests.js

```diff
@@ -18,8 +18,7 @@ function getSessionTokenId(sessionTokenHex) {
 }
 
 [{ version: '' }, { version: 'V2' }].forEach((testOptions) => {
-  describe(`#integration${testOptions.version} - remote password change`, function () {
-    this.timeout(60000);
+  describe(`#integration${testOptions.version} - #serial - remote password change`, function () {
```

Comment:
Same as other comment, if you know the reason this has to be done in serial, adding a comment would be helpful.